lib/t_web/channels/feed_channel.ex

```diff
@@ -424,8 +421,7 @@ defmodule TWeb.FeedChannel do
             profile: profile,
             screen_width: screen_width,
             inserted_at: inserted_at,
-            expiration_date: expiration_date,
-            exchanged_voice: false
```

Comment:
or rather this is could be the breaking change, since the client doesn't care about our DB but it does care about the immediate responses from backend

Reply:
is it ok to remove this two fields? how does the client handle their absence? or maybe it doesn't use them at all?

Reply:
only `exchanged_voice` removed here, yeah it's okay